fiftyone/core/map/mapper.py

```diff
@@ -168,8 +168,18 @@ def map_samples(
             Iterator[Tuple[bson.ObjectId, R]]: The sample ID and the result of
               the map function for the sample.
         """
+        try:
+            result = check_if_return_is_sample(sample_collection, map_fcn)
+        except ValueError as e:
+            if "is empty" in str(e):
+                logger.warning(
+                    "Sample collection is empty. No samples to map."
```

Comment:
I definitely don't want to log a warning about the collection being empty. None of our existing methods log warnings in this case. They are just no-ops. We should follow that pattern here for consistency.